examples/ABMC.yml

```diff
@@ -0,0 +1,14 @@
+abbreviation: ABMC
+departments:
+- name: American Battle Monuments Commission
+  submission_methods:
+  - submission_format: usps
+    name: Beatriz Collazo
+    title: FOIA Assistant
```

Comment:
I would find it a little odd if the addressee name was *only* listed under the address key. I like that a person's name gets its own key.